scripts/get-hosts.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# PYTHON_ARGCOMPLETE_OK
+
+import argparse
+import base64
+import os
+import pathlib
+import sys
+import textwrap
+import tomllib
+import urllib.request
+import uuid
+
+from urllib.error import HTTPError
+
+from app_common_python import json
+
+
+try:
+    import argcomplete
+except ImportError:
+    argcomplete = None
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--org-id", help="Org ID", required=True, type=int)
+    parser.add_argument("-e", "--environment", choices=["prod", "stage"], default="prod")
+    parser.add_argument("-n", "--display-name", help="Filter by display_name")
+    parser.add_argument("-l", "--limit", type=int, help="Maximum number of records to retrieve")
+    parser.add_argument("-s", "--scrub", action="store_true", help="Anonymize the data")
+
+    if argcomplete:
+        argcomplete.autocomplete(parser)
+
+    return parser.parse_args()
+
+
+def load_config():
+    config_path = pathlib.Path(__file__).parents[1] / ".config.toml"
```

Comment:
For others, format of the file is `.config.toml` file placed in the root of the repo is:
```
[environments]
prod = "prod_query_url"
stage = "stage_query_url"
```